index.Rmd

```diff
@@ -487,6 +488,58 @@ getdoge(c("wow", "suchhostility", "bemoreinclusive"))
 
 ***Now, back to your regular, cat-dominated tutorial.***
 
+## <a href="#dataNames" name="dataNames"/>#</a> Data from the web: names
+
+
+Besides pictures, R is also really good at scraping data from websites! Here, we'll look at the names of famous cats that are listed on Wikipedia. 
+
+First, we'll have to load some helpful packages.
+```{r scrapePackages, collapse=TRUE}
+library(XML)
+library(RCurl)
+```
+
+Each of these packages will help us do different things. The XML package has really helpful functions for turning websites into useable R data. However, the XML package doesn't play nice with https, so we'll have to use another package to tell R that it doesn't have to worry about security--that is, the RCurl package. 
+
+We have an object telling R where to go on the internet, so let's store the URL into an object: 
+```{r makeURL, collapse=TRUE}
+url <- "https://en.wikipedia.org/wiki/Popular_cat_names"
+```
+
+Wait a minute! This website looks like it uses https! That's a bit annoying, but we can work around it. If we want to scrape data from an https website, we'll have to tell R not to use the extra layer of security.
+
+Fortunately, the RCurl package has a function that lets you choose not to use https. Phew! Here, we'll tell R to copy the webpage into a new object, "wikiCatNamePage". 
+```{r makePage, collapse=TRUE}
+wikiCatNamePage <- getURL(url,.opts = list(ssl.verifypeer = FALSE) )
```

Comment:
You're right that this works to disable SSL verify peer, but I think for this basic R intro website, we can make this easier for people. In addition, I understand that it is not great to teach people to do this as it should not always be done. `httr` can handle this automatically. See example in main PR comments